crypto/message.go

```diff
@@ -223,6 +223,24 @@ func (msg *PGPMessage) GetArmoredWithCustomHeaders(comment, version string) (str
 	return armor.ArmorWithTypeAndCustomHeaders(msg.Data, constants.PGPMessageHeader, version, comment)
 }
 
+// getEncryptionKeyIds Returns the key IDs of the keys to which the session key is encrypted.
+func (msg *PGPMessage) getEncryptionKeyIDs() []uint64 {
+	packets := packet.NewReader(bytes.NewReader(msg.Data))
+	var err error
```

Comment:
Is it safe to ignore errors here?

Reply:
yes, it is same pattern as in SeparateKeyAndData.